test/e2e/alertmanager_helpers.go

```diff
@@ -109,6 +109,7 @@ func setupWebhookReceiver(t *testing.T, f *framework.Framework, namespace string
 								"--log.level=debug",
 								`--id.template={{ .CommonLabels.alertname }}_{{ .CommonLabels.namespace }}`,
 							},
+							SecurityContext: getSecurityContext(),
```

Comment:
I'm surprised that you can specify both `allowPrivilegeEscalation` and `capabilities` here and not have to do it in the container section 🤔 since `oc explain deployment.spec.template.spec.securityContext` doesn't mention them...

Reply:
`deployment.spec.template.spec.securityContext` is [PodSecurityContext v1 core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podsecuritycontext-v1-core), but this change uses `deployment.spec.template.spec.containers.securityContext` that is [SecurityContext v1 core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#securitycontext-v1-core), because:

- Some fields like `allowPrivilegeEscalation`, that appears in the warning, appear in SecurityContext but not in PodSecurityContext
- As documented in [PodSecurityContext v1 core](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#podsecuritycontext-v1-core), "field values of container.securityContext take precedence over field values of PodSecurityContext" so this way the change is more stable to future modifications